recipes/salad/stir-fry-cabbage.md

```diff
@@ -0,0 +1,58 @@
+# Stir-Fry Cabbage
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-29%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-0.5hr-red.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-VeryCheap-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+* **First cooked by:** [@heathercoraje](https://github.com/Heathercoraje)
+* **Date:** Week 11, 13 FACN2
+* **Author of Recipe**: [@heathercoraje](https://github.com/Heathercoraje)
+
+**The story behind this dish**
+The author of recipe often suffers from acid reflux and indigestion. Cabbage is known to help reduce stomach pain and intestinal Ulcers.
+[Read more](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1643665/). Because fresh cabbage juice isn't her favorite beverage, she came up with a brilliant idea to eat a large amount of cabbage and actually enjoy eating it.
+
+```
+return Stir-Fry Cabbage
+```
+
+## Ingredients
+
+A wok, vegetable oil (olive oil x), 1/2 of a large white cabbage, 3 pieces of 50g of slim Scallion (Green onion), 2 pieces of large dried chili pepper, a base sauce, 5 or more cloves of garlic.
+
+* Base sauce needs to be prepared boiled with following ingredients
+  * 4:5:1 ratio of Soy sauce, Oyster sauce and sugar.
+  * 5:5 ratio of Soy sauce and sugar for veggie friends.
```

Comment:
I would add soy sauce, oyster sauce and sugar to the ingredient list.

Then add how to make the sauce in the preparation section? :point_down:

Reply:
are these definitely the right ratios?

The first one is a `9:1` `sauce:sugar` ratio and the second is a `1:1` `sauce:sugar` ratio.

If they are correct, the second one should read `1:1` instead of `5:5` I think.

Reply:
Oh math-retarded gets caught here by a math-major. What I wanted to say is 1:1 (sugar and soy sauce) then for non-veggies, there are 3 components, soy sauce, oyster sauce and sugar. How would I describe this ration? @mattlub